util/logutil/inconsist/reporter.go

```diff
@@ -0,0 +1,264 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package inconsist
```

Comment:
The package name looks a bit scary.

Reply:
+1 there could be a better name.

Reply:
@ekexium do you have an advice name~? :D

Reply:
`consistency`, `inconsistency`, `data`, `dataerror`, `corrupt`?